components/Buttons/LinkButton/LinkButton.tsx

```diff
@@ -1,37 +1,40 @@
 import classNames from 'classnames';
-import { string, node, bool, oneOf } from 'prop-types';
 import Link from 'next/link';
 import OutboundLink from 'components/OutboundLink/OutboundLink';
 import styles from '../Button/Button.module.css';
 
-LinkButton.propTypes = {
-  // Only pass analytics event label if you're href is to an external website
-  analyticsEventLabel: string,
-  children: node.isRequired,
-  className: string,
-  'data-testid': string,
-  fullWidth: bool,
-  href: string.isRequired,
-  theme: oneOf(['primary', 'secondary']),
-};
-
-LinkButton.defaultProps = {
-  analyticsEventLabel: undefined,
-  className: undefined,
-  'data-testid': undefined,
-  fullWidth: false,
-  theme: 'primary',
-};
+export type LinkButtonProps = {
+  /**
+   * Url to be passed to the base anchor element.
+   */
+  href: string;
+  /**
+   * Only pass analytics event label if you're href is to an external website
+   */
+  analyticsEventLabel?: string;
+  /**
+   * Sets an id to the base element for testing.
+   */
+  'data-testid'?: string;
+  /**
+   * Forces the component's width as wide as its parent container's width.
+   */
+  fullWidth?: boolean;
+  /**
+   * Sets the button color theme.
+   */
+  theme?: 'primary' | 'secondary';
+} & React.ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
this actually isn't really true right? we don't pass through these props so we shouldn't make them allowed members of the type.

We just use `className` and `children` from this type.

Reply:
Ahh... Yea, I got the word "button" stuck in my head and forgot the base component was not a button in this case.